pkg/operators/v1alpha1/clusterserviceversion_types.go

```diff
@@ -274,8 +275,10 @@ type APIServiceDefinitions struct {
 // that can manage apps for a given version.
 // +k8s:openapi-gen=true
 type ClusterServiceVersionSpec struct {
-	InstallStrategy           NamedInstallStrategy      `json:"install"`
-	Version                   version.OperatorVersion   `json:"version,omitempty"`
+	InstallStrategy NamedInstallStrategy    `json:"install"`
+	Version         version.OperatorVersion `json:"version,omitempty"`
+	// +optional
+	Release                   release.OperatorRelease   `json:"release,omitzero"`
```

Comment:
```suggestion
	Release                   *release.OperatorRelease   `json:"release,omitempty"`
```

Reply:
Make it a pointer and use omitempty instead of omitzero?

Reply:
Other than that, this reduces the number of required checks.  With a pointer, it is:
1. nil := no release
2. non-nil and empty := no release
3. non-nil and non-empty := has release

with the instance, it's just 
1. empty := no release
2. non-empty := has release


For more discussion on the feature, check out [this article](https://www.jvt.me/posts/2025/02/12/go-omitzero-124/).

Reply:
omitzero is new in go1.24, so our conventions docs do not address it.